editor/editor_settings.cpp

```diff
@@ -1373,19 +1373,46 @@ float EditorSettings::get_auto_display_scale() const {
 
 // Shortcuts
 
-void EditorSettings::add_shortcut(const String &p_name, const Ref<Shortcut> &p_shortcut) {
-	shortcuts[p_name] = p_shortcut;
+void EditorSettings::_add_shortcut(const String &p_path, Ref<Shortcut> p_shortcut) {
```

Comment:
It's been a while since I have written C++, but I notice that the & was removed along with the const, which doesn't make sense to me, and I wanted to make sure that is intentional.